src/main/resources/application.conf

```diff
@@ -36,6 +36,12 @@ workflow-options {
 
   // AES-256 key to use to encrypt the values in `encrypted-fields`
   base64-encryption-key: "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA="
+
+  // Directory where to write per workflow logs
+  workflow-log-dir: "cromwell-workflow-logs"
+
+  // When true, per workflow logs will be deleted after copying
+  workflow-log-temporary: true
```

Comment:
Fair point... especially if you're using the command-line runner.

I'm just thinking... Apache or MySQL do by default?  Well all server software I can think of logs to a file by default.  Perhaps that should be Cromwell's default behavior when in server mode?